package/libpiksi/libpiksi/src/util.c

```diff
@@ -184,7 +188,12 @@ u16 sbp_sender_id_get(void)
   char sbp_sender_id_string[32];
   if (file_read_string(SBP_SENDER_ID_FILE_PATH, sbp_sender_id_string, sizeof(sbp_sender_id_string))
       == 0) {
-    sbp_sender_id = strtoul(sbp_sender_id_string, NULL, 10);
+    unsigned long ul_value = strtoul(sbp_sender_id_string, NULL, 10);
+    if (ul_value > ((u16) ~0u)) {
```

Comment:
is this the same as `u1_value >= UINT16_MAX` ?

Reply:
Yes, will fix